content/pt-br/_collections/navigations/footer.md

```diff
@@ -0,0 +1,55 @@
+---
+links:
+  - title: 'Sobre'
+    items:
+      - title: 'Fale conosco'
+        href: 'mailto:hello@nuxtjs.com'
+      - title: 'Enterprise support'
+        to: '/support'
+      - title: 'Empresa NuxtLabs'
+        href: 'https://nuxtlabs.com/'
+      - title: 'Software de código aberto'
+        href: 'https://github.com/nuxt'
+      - title: 'Parcerias'
+        to: '/partners'
+      - title: 'Telemetria'
+        href: 'https://github.com/nuxt/telemetry'
+  - title: 'Ecossistema'
+    items:
+      - title: 'Anúncios'
+        to: '/announcements'
+      - title: 'Contibua'
```

Comment:
`Contribuar`?

Reply:
`Contibua` is wrong, I changed it to `Contribuir`.